alert/jp.go

```diff
@@ -0,0 +1,3326 @@
+package alert
+
+// Alert has XCERT alert information
+type Alert struct {
+	URL   string
+	Title string
+	Team  string
+}
+
+// JPCERT alerts
```

Comment:
__[Golint]__ <sub>posted by Sider</sub>

comment on exported var AlertDictJP should be of the form "AlertDictJP ..."